scripts/add-this-linux.sh

```diff
@@ -10,7 +10,7 @@ while true; do
 done
 
 mv meta-linux meta
-cp meta /usr/bin
+cp -r meta /usr/bin
```

Comment:
this requires me to run with sudo,
also it seem there you have conflicts,
always start the work with fresh master/main brunch copy or update your branch with main/master to prevent conflicts,
as you can see in main branch I have refactored and moved some lines in wget to directly pull into destination file